### DISCUSSION TITLE
How to move assets

### DISCUSSION THREAD

**Discussion by sivner-figbus** [requarks/wiki#3002] [Help / Questions] [answered]
I'm logged in to a wiki.js instance as an administrator.

I want to move some uploaded assets to a different folder, but it won't let me. The "move" function (along with "properties", "preview", "edit", and "optimize") is greyed out.

What am I doing wrong, please?

![image](https://user-images.githubusercontent.com/67953485/106510550-fb754180-64c6-11eb-8611-3c78f736fff4.png)

**Comment by NGPixel** [comment, marked as answer]
This is not possible yet. Coming in a future release.

**Reply by sivner-figbus** [reply]
Thank you.

I wasted a lot of time trying to understand what was wrong with my groups/users/permissions set up, because the menu option was there. 

I didn't pay anything for this software, so I can't complain, but it was frustrating to have a greyed-out option rather than a "not implemented" note, or just not have the option.
